ddtrace/tracer/metrics.go

```diff
@@ -93,8 +93,26 @@ func (t *tracer) reportHealthMetricsAtInterval(interval time.Duration) {
 	for {
 		select {
 		case <-ticker.C:
-			t.statsd.Count("datadog.tracer.spans_started", int64(atomic.SwapUint32(&t.spansStarted, 0)), nil, 1)
-			t.statsd.Count("datadog.tracer.spans_finished", int64(atomic.SwapUint32(&t.spansFinished, 0)), nil, 1)
+			// if there are started spans, report the number of spans with their integration, then
+			// reset the count
+			// the Count() function reports the total number of event occurrences in one time interval. We reset
+			// our count to 0 regardless of if Count succeeded to cleanup before the next interval.
+			t.spansStarted.Range(func(key string, value *atomic.Int64) bool {
+				v := value.Swap(0)
```

Comment:
```suggestion
				old := value.Swap(0)
```
nit: the name of the variable could it easier to reason about this piece code if one is not familiar with the `atomic` package and its operations.